tests/configuration/pages/pmmInventoryPage.js

```diff
@@ -55,13 +62,9 @@ module.exports = {
     I.scrollPageToBottom();
   },
 
-  async openAgents() {
-    await I.waitForVisible(this.fields.agentsLink, 20);
-    I.click(this.fields.agentsLink);
-    I.waitForElement(this.fields.pmmAgentLocator, 60);
+  async openAgents(serviceId) {
+    I.amOnPage(`graph/inventory/services/${serviceId.split('/')[2]}/agents`);
     await this.changeRowsPerPage(100);
```

Comment:
Could we move the "hidden" step out from the method to a test body as it's does not corresponds the method's name. Or rename this method into `openAgentsAndSet100rowsPerPage()` ?